fonts.css

```diff
@@ -2,127 +2,147 @@
     font-weight: bold;
     font-style: italic;
+    font-display: swap;
 }
 
 @font-face {
     font-family: "Qanelas Soft";
     src: url('./static/fonts/qanelas-soft/QanelasSoftExtraBold.otf') format('opentype');
     font-weight: 800;
+    font-display: swap;
 }
 
 @font-face {
     font-family: "Qanelas Soft";
     src: url('./static/fonts/qanelas-soft/QanelasSoftExtraBoldItalic.otf') format('opentype');
     font-weight: 800;
     font-style: italic;
+    font-display: swap;
 }
 
 @font-face {
     font-family: "Qanelas Soft";
     src: url('./static/fonts/qanelas-soft/QanelasSoftHeavy.otf') format('opentype');
     font-weight: 900;
+    font-display: swap;
 }
 
 @font-face {
     font-family: "Qanelas Soft";
     src: url('./static/fonts/qanelas-soft/QanelasSoftHeavyItalic.otf') format('opentype');
     font-weight: 900;
     font-style: italic;
+    font-display: swap;
 }
 
 @font-face {
     font-family: "Qanelas Soft";
     src: url('./static/fonts/qanelas-soft/QanelasSoftLight.otf') format('opentype');
     font-weight: 300;
+    font-display: swap;
 }
 
 @font-face {
     font-family: "Qanelas Soft";
     src: url('./static/fonts/qanelas-soft/QanelasSoftLightItalic.otf') format('opentype');
     font-weight: 300;
     font-style: italic;
+    font-display: swap;
 }
 
 @font-face {
     font-family: "Qanelas Soft";
     src: url('./static/fonts/qanelas-soft/QanelasSoftMedium.otf') format('opentype');
     font-weight: 500;
+    font-display: swap;
 }
 
 @font-face {
     font-family: "Qanelas Soft";
     src: url('./static/fonts/qanelas-soft/QanelasSoftMediumItalic.otf') format('opentype');
     font-weight: 500;
     font-style: italic;
+    font-display: swap;
 }
 
 @font-face {
     font-family: "Qanelas Soft";
     src: url('./static/fonts/qanelas-soft/QanelasSoftRegular.otf') format('opentype');
     font-weight: 400;
+    font-display: swap;
 }
 
 @font-face {
     font-family: "Qanelas Soft";
     src: url('./static/fonts/qanelas-soft/QanelasSoftRegularItalic.otf') format('opentype');
     font-weight: 400;
     font-style: italic;
+    font-display: swap;
 }
 
 @font-face {
     font-family: "Qanelas Soft";
     src: url('./static/fonts/qanelas-soft/QanelasSoftSemiBold.otf') format('opentype');
     font-weight: 600;
+    font-display: swap;
 }
 
 @font-face {
     font-family: "Qanelas Soft";
     src: url('./static/fonts/qanelas-soft/QanelasSoftSemiBoldItalic.otf') format('opentype');
     font-weight: 600;
     font-style: italic;
+    font-display: swap;
 }
 
 @font-face {
     font-family: "Qanelas Soft";
     src: url('./static/fonts/qanelas-soft/QanelasSoftThin.otf') format('opentype');
     font-weight: 100;
+    font-display: swap;
 }
 
 @font-face {
     font-family: "Qanelas Soft";
     src: url('./static/fonts/qanelas-soft/QanelasSoftThinItalic.otf') format('opentype');
     font-weight: 100;
     font-style: italic;
+    font-display: swap;
```

Comment:
@Abhishek-kumar09 , @Chadha93  we might should add `font-display` in Meshery extensions.